.github/workflows/release.yml

```diff
@@ -130,40 +127,21 @@ jobs:
         with:
           name: packages
           path: packages
-      - name: Install NuGet packages and dotnet new templates
-        working-directory: packages
+      - name: Configure NuGet sources
         run: |
-          dotnet nuget push *.nupkg --source ~/.nuget/packages
-          dotnet new install IceRpc.Templates*.nupkg
-        if: runner.os == 'macOS' || runner.os == 'Linux'
-        shell: bash
-      - name: Install NuGet packages and dotnet new templates
+          dotnet new nugetconfig -o ${{ github.workspace }}
+          dotnet nuget add source --configfile ${{ github.workspace }}/nuget.config -n "Local Packages" ${{ github.workspace }}/packages
+        shell: pwsh
+      - name: Install dotnet new templates
         working-directory: packages
-        run: |
-          dotnet nuget push *.nupkg --source $env:USERPROFILE/.nuget/packages
-          dotnet new install IceRpc.Templates*.nupkg
-        if: runner.os == 'Windows'
-        shell: powershell
-      - name: Set version from tag
-        run: |
-          VERSION=${GITHUB_REF#refs/tags/v}
-          echo "VERSION=$VERSION" >> $GITHUB_ENV
-        if: runner.os == 'macOS' || runner.os == 'Linux'
-      - name: Set version from tag
         run: |
           $version = $env:GITHUB_REF -replace '^refs/tags/v',''
-          echo "VERSION=$version" >> $env:GITHUB_ENV
-        if: runner.os == 'Windows'
-      - name: 🔨 Build Examples
-        run: |
-          for solution in examples/*/*/*.sln; do dotnet build "$solution"; done
-        if: runner.os == 'macOS' || runner.os == 'Linux'
-        shell: bash
+          dotnet new install IceRpc.Templates::$version
```

Comment:
This install the project templates from the local source, we use `::$version` to ensure we are testing the new version of the templates.